tests/mpls_input/appl1_db.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
are these files required? Can't we use the ones in "asic0/xx" and "asic1/xx"

Reply:
no, will remove them